app/javascript/react/screens/App/Settings/__tests__/Settings.test.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Spinner } from 'patternfly-react';
+import { shallow } from 'enzyme';
+import { Settings } from '../Settings';
+import { servers, settings } from '../settings.fixures';
+import { getFormValuesFromApiSettings } from '../helpers';
+
+describe('Settings component', () => {
+  const defaultFormValues = getFormValuesFromApiSettings(settings);
+  const baseProps = {
+    fetchServersAction: jest.fn(),
+    fetchSettingsAction: jest.fn(),
+    patchSettingsAction: jest.fn(),
```

Comment:
While it doesn't affect the tests here, it's not a bad idea to make sure that each test case gets a fresh set of mocks by doing something like 

```js
let myMock;
beforeEach(() => {
  myMock = jest.fn();
});
```
Jest mock functions keep track of their number of calls and results.  See [second code snippet here](https://jestjs.io/docs/en/mock-functions).

You won't be able to store them in a `baseProps` object any more, unfortunately, and since you're not checking those values in these tests, I'll leave it up to you whether or not to leave this as is